lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -210,6 +224,21 @@ def initialize
     def configure(conf)
       super
 
+      # If monitoring is enabled, register metrics in the default registry
+      # and store metric objects for future use.
+      if @monitoring_enabled
+        registry = Monitoring::MonitoringRegistryFactory.create @monitoring_type
+        @successful_requests_count = registry.counter(
+          :stackdriver_successful_requests_count,
+          'A number of successful requests to the Stackdriver Logging API')
+        @failed_requests_count = registry.counter(
+          :stackdriver_failed_requests_count,
+          'A number of failed requests to the Stackdriver Logging API')
+        @log_entries_count = registry.counter(
+          :stackdriver_log_entries_count,
+          'A number of log entries sent to the Stackdriver Logging API')
```

Comment:
Well, that ARE sent, just some of them were ingested and some -- weren't

But ack, I'll change